pkg/builder/in_memory_build_queue_test.go

```diff
@@ -158,34 +158,6 @@ func TestInMemoryBuildQueueExecuteBadRequest(t *testing.T) {
 		require.Equal(t, err, status.Error(codes.FailedPrecondition, "Failed to obtain command: Blob not found"))
 	})
 
-	// Command does not contain any playform requirements.
```

Comment:
Ha, typo in there!

Instead of removing this test, could we either change it (or add a new test) to validate the new behaviour?

Reply:
I guess it would be best to move out from under the `BadRequest` test? Or maybe it still fits there as a missing platform message could still be considered a bad request?

Reply:
Most of the tests for the correct cases below use the same platform properties. I think it's acceptable if you just took one of those tests and modified it to use a nil platform message.

Reply:
Do you mean just take any one of the existing tests, such as `TestInMemoryBuildQueuePurgeStaleWorkersAndQueues`, and change the platform message to be nil? Or to create a new test which beaves similarly to one of the other tests but with a nil platform?

I tried leaving it in the same place and checking that the error is that no workers exist rather than the platform isn't specified, but that is probably not a great idea as it's not isolating one the part of the code being tested.

Reply:
The first. Just adjust one of the existing ones. :-)